main/monmon.rb

```diff
@@ -3,9 +3,14 @@
 
 options = {file: "input.csv"}
 OptionParser.new do |opts|
-  opts.on("-f name", "--file=name") do |v|
+  opts.banner = "Usage:   monmon.rb [options]"
+  opts.on("-f name", "--file=name", "Print this to load other file") do |v|
```

Comment:
This description is confusing. Maybe `"Input file in CSV format"` would be better?